validation/services/user.js

```diff
@@ -0,0 +1,19 @@
+const allowedUserFieldsForUpdate = {
+  photo: true,
+  firstName: true,
+  lastName: true,
+  address: {
+    country: true,
+    city: true
+  },
+  education: true,
+  categories: true,
+  totalReviews: true,
```

Comment:
pardon me, do i still see those fields here? totalReviews and averageRating or my GIT is acting out? they must be removed